site/class-1-bar/vincentkempers/index.js

```diff
@@ -0,0 +1,74 @@
+
```

Comment:
Great, but could you add a citation to the original work you based this chart on, in the HTML and JS?